web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/avslutt/AvsluttRevurderingRequestJson.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.su.se.bakover.web.routes.revurdering.avslutt
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import io.ktor.http.HttpStatusCode.Companion.BadRequest
+import no.nav.su.se.bakover.domain.brev.Brevvalg
+import no.nav.su.se.bakover.web.Resultat
+import no.nav.su.se.bakover.web.errorJson
+import no.nav.su.se.bakover.web.routes.brev.BrevvalgJson
+
+/**
+ * Dersom
+ *
+ * @param fritekst null dersom SKAL_IKKE_SENDE_BREV og utfylt dersom SKAL_SENDE_BREV_MED_FRITEKST.
+ * @param begrunnelse Skal inneholde informasjon om hvorfor saksbehandler valgte å avslutte revurderingen. Dersom saksbehandler velger at det ikke skal sendes brev, skal dette og begrunnes her.
+ */
+internal data class AvsluttRevurderingRequestJson(
```

Comment:
Slipper vi unna med å dele denne requesten for flere forskjellige formål? Begrunne avslutting og begrunne utsendelse/ikke av brev?

Reply:
Det var oppe til diskusjon, men vi ville ikke pålegge to beskrivelsesfelter på saksbehandler (selvom det stakk litt kjepper i datamodellen). Jeg tenker denne diskusjonen dukker opp igjen når fagsiden får testet det litt.

Jeg har på følelsen av at forskjellig brev i forskjellig situasjoner kommer til å ha litt forskjellige krav til brevvalg, selvom jeg håpet i disse baner selv.